src/controllerclientimpl.cpp

```diff
@@ -123,6 +123,16 @@ ControllerClientImpl::ControllerClientImpl(const std::string& usernamepassword,
         _basewebdavuri = str(boost::format("%su/%s/")%_baseuri%_username);
     }
 
+    _mujinControllerIp = _baseuri.substr(_baseuri.find("//")+2, _baseuri.size());
+    _mujinControllerIp.erase(remove(_mujinControllerIp.begin(), _mujinControllerIp.end(), '/'), _mujinControllerIp.end());  // Remove / from string
+    // Remove port
+    int idx = _mujinControllerIp.find(':');
+    if (idx != std::string::npos) {
+        _mujinControllerIp = _mujinControllerIp.substr(0, idx);
+    }
+    boost::shared_ptr<zmq::context_t> zmqcontext(new zmq::context_t(1));
+    InitializeZMQ(7801, zmqcontext);
```

Comment:
@felixvd well, just in case, could you tell me where is 7801 used? (eg webstack branch) /cc @woswos 

https://github.com/mujin/mujinwebstackclientpy/pull/10/files#diff-545194d7accdcca89d68f53c930f432ed36c8ac025e2dc85d017eb1f971505abR183 too

Reply:
guessing zmqServer20230320

Reply:
done (but please show how you are testing this change, really)